internal/parser/node_groups.go

```diff
@@ -17,9 +17,28 @@ type inputNodeGroup struct {
 	NodesType     any `default:"PC"`
 	Networks      []string
 
-	MinSpeed any `default:"0"`
-	MaxSpeed any `default:"1"`
-	MaxPause any `default:"0"`
+	MinSpeed              any `default:"1"`
```

Comment:
is somewhere described what the values do?

Reply:
The meanings are described in the [wiki](https://github.com/stg-tud/bp2022_netlab/wiki/Node-Groups#randomwaypoint). But that's a good point, I forgot to remove the unused ones, now that I removed the additional movement patterns.

Reply:
Should be fixed now